.editorconfig

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2022-present, IO Visor Project
+# SPDX-License-Identifier: Apache-2.0
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = crlf
+charset = utf-8
+trim_trailing_whitespace = false
+insert_final_newline = false
```

Comment:
I have a weak personal preference to set both of these to true.  But if you have a strong opinion not to do this then feel free to keep it as it is.